src/Microsoft.Identity.Client/Core/Constants.cs

```diff
@@ -14,6 +14,8 @@ internal static class Constants
 
         public const string UapWEBRedirectUri = "https://sso"; // only ADAL supports WEB
         public const string DefaultRedirectUri = "urn:ietf:wg:oauth:2.0:oob";
+        public const string DefaultDesktopRedirectUri = "https://login.microsoftonline.com/common/oauth2/nativeclient";
+        public const string DefaultNetCoreRedirectUri = "https://localhost";
```

Comment:
Should probably not be named netcore. Should be named LocalhostRedirectUri imo. And the one above should be renamed to NativeClientRedirectUri